lib/datashift_spree/product_populator.rb

```diff
@@ -37,8 +47,45 @@ def prepare_and_assign_method_binding(method_binding, record, data)
         add_options_variants
 
       elsif(method_binding.operator?('price') )
-
         product_load_object.price = value.to_f
+        if !product_load_object.new_record?
+          product_load_object.save
+        end
+
+      elsif(method_binding.operator?('cost_price'))
+        product_load_object.cost_price = value.to_f
+        if !product_load_object.new_record?
+          product_load_object.save
+        end
+
+      elsif(method_binding.operator?('shipping_category'))
+        if !product_load_object.new_record?
+          product_load_object.shipping_category.name = value
```

Comment:
change this product_load_object.shipping_category.name = value
to 
product_load_object&.shipping_category&.name = value